private/buf/bufcli/flags_args.go

```diff
@@ -277,11 +277,11 @@ func PackageVersionLongDescription(registryName, commandName, examplePlugin stri
 Examples:
 
 Get the version of the eliza module and the %s plugin for use with %s.
-    $ buf alpha sdk %s --module=buf.build/connectrpc/eliza --plugin=%s
+    $ buf registry sdk %s --module=buf.build/connectrpc/eliza --plugin=%s
         v1.7.0-20230609151053-e682db0d9918.1
 
 Use a specific module version and plugin version.
-    $ buf alpha sdk %s --module=buf.build/connectrpc/eliza:233fca715f49425581ec0a1b660be886 --plugin=%s:v1.0.0
+    $ buf registry sdk %s --module=buf.build/connectrpc/eliza:233fca715f49425581ec0a1b660be886 --plugin=%s:v1.0.0
```

Comment:
If we want the examples to be realistic, we'd want to pass down a plugin version here too as not every plugin has a v1.0.0 release.